### PR TITLE
Fixed class= to ClassName = in Navbar and Footer

### PR DESCRIPTION
**Pull request by Vijay-Gunwant**
This is a basic change to the code base. Hence I fixed it also and added the contact us feature with a email(specified)

### CODE REVIEW

**Comment by AdarshRawat1** [comment]
Use this email instead  -  openera@myyahoo.com

File: .vs/Contribution/FileContentIndex/b4d69a9c-1573-489b-8223-9fbc4ae7b953.vsidx
Comment (AdarshRawat1) [inline .vs/Contribution/FileContentIndex/b4d69a9c-1573-489b-8223-9fbc4ae7b953.vsidx:1]: Avoid including binary files in the PR, as they consume storage space and need to be compiled on each user's computer anyway, making it unnecessary.

File: .vs/ProjectSettings.json
Comment (AdarshRawat1) [inline .vs/ProjectSettings.json:1]: Make sure to avoid tracking your VS Code configuration files.

File: .vs/slnx.sqlite
Comment (AdarshRawat1) [inline .vs/slnx.sqlite:1]: BIN file

File: pages/components/Footer.js [hunk 1]
```diff
@@ -2,27 +2,27 @@ import React from 'react'
...
                     </a>
-                    <ul class="flex flex-wrap items-center gap-6 mb-6 text-lg text-gray-800 sm:mb-0">
+                    <ul className="flex flex-wrap items-center gap-6 mb-6 text-lg text-gray-800 sm:mb-0">
                         <li>
-                            <a href="#" class="hover:underline hover:text-blue-800">About</a>
+                            <a href="#" className="hover:underline hover:text-blue-800">About</a>
                         </li>
                         <li>
-                            <a href="#" class="hover:underline hover:text-blue-800">Contact</a>
+                            <a href="mailto:abc@gmail.com" className="hover:underline hover:text-blue-800">Contact</a>
```
Comment (AdarshRawat1) [inline pages/components/Footer.js]: use - openera@myyahoo.com

File: .vs/VSWorkspaceState.json
Comment (AdarshRawat1) [inline .vs/VSWorkspaceState.json:1]: Avoid the tracking of configuration files.

File: .vs/Contribution/v17/.wsuo
Comment (AdarshRawat1) [inline .vs/Contribution/v17/.wsuo:1]: BIN file

**Review by AdarshRawat1 — CHANGES_REQUESTED** [review]

File: .vs/Contribution/FileContentIndex/b4d69a9c-1573-489b-8223-9fbc4ae7b953.vsidx
Comment (Vijay-Gunwant) [reply .vs/Contribution/FileContentIndex/b4d69a9c-1573-489b-8223-9fbc4ae7b953.vsidx:1]: Done I added these files to the GitIgnore file

File: .vs/ProjectSettings.json
Comment (Vijay-Gunwant) [reply .vs/ProjectSettings.json:1]: Done

File: .vs/slnx.sqlite
Comment (Vijay-Gunwant) [reply .vs/slnx.sqlite:1]: Done

File: pages/components/Footer.js (hunk 1 above)
Comment (Vijay-Gunwant) [reply pages/components/Footer.js]: Done

**Comment by AdarshRawat1** [comment]
@Vijay-Gunwant  kindly address other requested changes . ( Don't track- VS code configuration/ BIN file )

"tracking" a file means keeping a record of changes. To stop Git from tracking a file, add it to `.gitignore`.To stop tracking an already tracked file, use git `rm --cached <file>`.

I'll update the `.gitignore` file so that you don't have to do it from next time.

**Comment by Vijay-Gunwant** [comment]
> @Vijay-Gunwant kindly address other requested changes . ( Don't track- VS code configuration/ BIN file )
> 
> "tracking" a file means keeping a record of changes. To stop Git from tracking a file, add it to `.gitignore`.To stop tracking an already tracked file, use git `rm --cached <file>`.
> 
> I'll update the `.gitignore` file so that you don't have to do it from next time.

I did update the gitignore file in my last commit including the .vs folder in the gitingore file . As of stopping github to track a file i was not aware about it which i will do in my next commit.Thanks to yout such valuable and calm response

**Comment by AdarshRawat1** [comment]
> I did update the gitignore file in my last commit including the .vs folder in the gitingore file. 

Thank you @Vijay-Gunwant for addressing the requested changes.  (Kindly mark the requested changes as resolved). 
> As of stopping github to track a file i was not aware about it which i will do in my next commit.

Now that you have added the VS Code directory in `.gitignore` they won't be tracked so no worries. I also updated the `.gitignore`  in main branch so make sure to sync your fork. 
> Thanks to yout such valuable and calm response

Anytime!

File: .vs/Contribution/v17/.wsuo
Comment (Vijay-Gunwant) [reply .vs/Contribution/v17/.wsuo:1]: done

File: .vs/VSWorkspaceState.json
Comment (Vijay-Gunwant) [reply .vs/VSWorkspaceState.json:1]: done

**Comment by Vijay-Gunwant** [comment]
> > I did update the gitignore file in my last commit including the .vs folder in the gitingore file.
> 
> Thank you @Vijay-Gunwant for addressing the requested changes. (Kindly mark the requested changes as resolved).
> 
> > As of stopping github to track a file i was not aware about it which i will do in my next commit.
> 
> Now that you have added the VS Code directory in `.gitignore` they won't be tracked so no worries. I also updated the `.gitignore` in main branch so make sure to sync your fork.
> 
> > Thanks to yout such valuable and calm response
> 
> Anytime!

I have marked them and would now you merge this pull request?

**Pull request merged**